src/core/components/Header/Header.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ChangeEvent, FC, useState } from "react";
+import LinkItem from "@/common/components/LInkItem/LinkItem";
+import Category from "./components/Category/Category";
+import Selector from "./components/Selector/Selector";
+import logo from "@/assets/images/logo.svg";
+import arrowDown from "@/assets/images/arrow_down.svg";
+import searchIcon from "@/assets/images/search.svg";
+import closeIcon from "@/assets/images/close.svg";
+import userIcon from "@/assets/images/user.svg";
+import cartIcon from "@/assets/images/basket.svg";
+import "./Header.scss";
+
+const categories = [
```

Comment:
Shouldn't be there. Would be better to create mocks folder

Reply:
Fixed - create mock data folder